### PR TITLE
DOC: Fix testing macro name

### PR DESCRIPTION
**Pull request by jhlegarreta**
Fix testing macro name: `TEST_SET_GET_OBJECT` does not exist as a testing
macro (or as any other macro in ITK).

### CODE REVIEW

**Comment by jhlegarreta** [comment]
Looks an oversight, but I'm curious if this existed at some point in time. Anyone?

File: SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex [hunk 1]
```diff
@@ -3571,7 +3571,7 @@ \subsection{Code Layout in Tests}
 \item \code{foo} class instantiation and basic object checks (e.g.
 \code{EXERCISE\_BASIC\_OBJECT\_METHODS}).
 \item \code{foo} class properties' input argument read and test (e.g. using the
-macros in \code{itkTestingMacro.h}, such as \code{TEST\_SET\_GET\_OBJECT}, etc.).
```
Comment (dzenanz) [inline SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex]: These macros are now prefixed by `ITK_`, so `ITK_TEST_SET_GET_OBJECT` etc. See https://github.com/InsightSoftwareConsortium/ITK/commit/dbb1532dd8fc1d0c263a71ded75944a30a890a6c.

Comment (dzenanz) [reply SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex]: Now I see you that are aware of the rename.

**Comment by jhlegarreta** [comment]
Rebased on master to solve conflicts.

**Pull request merged**